crates/bevy_math/src/bounding/bounded3d/primitive_impls.rs

```diff
@@ -299,6 +299,23 @@ impl Bounded3d for Torus {
         BoundingSphere::new(translation, self.outer_radius())
     }
 }
+impl Bounded3d for Ramp {
+    fn aabb_3d(&self, translation: Vec3, rotation: Quat) -> Aabb3d {
+        let points = [
+            self.half_size,
+            self.half_size * Vec3::new(1.0, -1.0, 1.0),
+            self.half_size * Vec3::new(1.0, -1.0, -1.0),
+            self.half_size * Vec3::new(-1.0, 1.0, 1.0),
+            self.half_size * Vec3::new(-1.0, -1.0, 1.0),
+            self.half_size * Vec3::NEG_ONE,
+        ];
+        Aabb3d::from_point_cloud(translation, rotation, points.into_iter())
+    }
+
+    fn bounding_sphere(&self, translation: Vec3, _rotation: Quat) -> BoundingSphere {
+        BoundingSphere::new(translation, self.half_size.length())
```

Comment:
I'm pretty sure this isn't guaranteed to be the tightest possible bounding sphere, but I won't block over that

Reply:
That is actually the tightest possible fit. The longest distance through the ramp would be from one of the lower corners of the "ramp-face" to the opposite corner of the "ramp-face". This distance corresponds to `2. * self.halfsize.length()` and is a lower bound for the diameter of the bounding sphere, placing the center of that sphere at `0` (in ramp-coordinates). As all other points are within this sphere, it is the smallest possible.

Reply:
Sorry, you're right; my initial intuition of where the center should be was wrong for some reason